### DISCUSSION TITLE
Keybinds not working?

### DISCUSSION THREAD

**Discussion by mikinis111** [regolith-linux/regolith-desktop#712] [Q&A]
I'm sorry if my question is stupid, I'm a complete beginner in linux, but what could be the reason of shortcuts not working? I can use `super + enter` but basically only this. I can't open an application launcher, neither close an app. Maybe I don't have necessary packages? I'm using linux mint 21. The problem appeared right after I finished installation of  the Regolith.

**Comment by cclamb** [comment]
Same for me. I have all the packages you suggest installed. I can't use keybindings for anything - VMWare Fusion, Ubuntu 22.04 LTS, Regolith Desktop 3.2. The VM is configured to pass all keystrokes to the VM, not the host. Raw i3 works just fine - all keybindings work as expected. I also tried to manually set the $mod key in .confg/regolith3/i3/config vai "set $mod Mod4" and that failed as well.

Installed regolith packages:

ayu-theme/unknown,now 0.2.2-1-1regolith-noble arm64 [installed,automatic]
fonts-nerd-font-bitstreamverasansmono/unknown,now 2.1.0-2-1regolith-noble all [installed,automatic]
i3-next-workspace/unknown,now 1.0.3-1regolith-noble arm64 [installed,automatic]
i3-swap-focus/unknown,now 0.4.3-1regolith-noble arm64 [installed,automatic]
i3xrocks-cpu-usage/unknown,now 5.5.1-1regolith-noble arm64 [installed,automatic]
i3xrocks-net-traffic/unknown,now 5.5.1-1regolith-noble arm64 [installed,automatic]
i3xrocks-time/unknown,now 5.5.1-1regolith-noble arm64 [installed,automatic]
i3xrocks/unknown,now 1.3.5-1-1regolith-noble arm64 [installed,automatic]
ilia/unknown,now 0.14.1-1regolith-noble arm64 [installed,automatic]
picom/unknown,now 11.2.2-1regolith-noble arm64 [installed,automatic]
regolith-compositor-picom-glx/unknown,now 1.3.0-1regolith-noble arm64 [installed,automatic]
regolith-control-center/unknown,now 1.46.0-3-1regolith-noble arm64 [installed,automatic]
regolith-default-settings/unknown,now 2.0.4-1regolith-noble arm64 [installed,automatic]
regolith-desktop/unknown,now 4.8.0-1regolith-noble arm64 [installed]
regolith-ftue/unknown,now 2.2.0-1regolith-noble arm64 [installed,automatic]
regolith-i3-compositor/unknown,now 4.11.6-1regolith-noble arm64 [installed,automatic]
regolith-i3-control-center-regolith/unknown,now 4.11.6-1regolith-noble arm64 [installed,automatic]
regolith-i3-dbus-activation/unknown,now 4.11.6-1regolith-noble arm64 [installed,automatic]
regolith-i3-default-style/unknown,now 4.11.6-1regolith-noble arm64 [installed,automatic]
regolith-i3-gaps/unknown,now 4.11.6-1regolith-noble arm64 [installed,automatic]
regolith-i3-i3xrocks/unknown,now 4.11.6-1regolith-noble arm64 [installed,automatic]
regolith-i3-ilia/unknown,now 4.11.6-1regolith-noble arm64 [installed,automatic]
regolith-i3-root-config/unknown,now 4.11.6-1regolith-noble arm64 [installed,automatic]
regolith-i3-session/unknown,now 4.11.6-1regolith-noble arm64 [installed,automatic]
regolith-i3-unclutter/unknown,now 4.11.6-1regolith-noble arm64 [installed,automatic]
regolith-i3xrocks-config/unknown,now 5.5.1-1regolith-noble arm64 [installed,automatic]
regolith-look-default-loader/unknown,now 0.8.2-1regolith-noble arm64 [installed,automatic]
regolith-look-default/unknown,now 0.8.2-1regolith-noble arm64 [installed,automatic]
regolith-look-lascaille/unknown,now 0.9.0-1regolith-noble arm64 [installed]
regolith-rofication/unknown,now 1.5.0-1regolith-noble arm64 [installed,automatic]
regolith-session-common/unknown,now 1.1.12-1regolith-noble arm64 [installed,automatic]
regolith-session-flashback-ext/unknown,now 1.1.12-1regolith-noble arm64 [installed,automatic]
regolith-session-flashback/unknown,now 1.1.12-1regolith-noble arm64 [installed]
regolith-unclutter-xfixes/unknown,now 1.5-2-1regolith-noble arm64 [installed,automatic]
regolith-wm-base-launchers/unknown,now 4.11.6-1regolith-noble arm64 [installed]
regolith-wm-config/unknown,now 4.11.6-1regolith-noble arm64 [installed,automatic]
regolith-wm-ftue/unknown,now 4.11.6-1regolith-noble arm64 [installed,automatic]
regolith-wm-navigation/unknown,now 4.11.6-1regolith-noble arm64 [installed]
regolith-wm-networkmanager/unknown,now 4.11.6-1regolith-noble arm64 [installed,automatic]
regolith-wm-resize/unknown,now 4.11.6-1regolith-noble arm64 [installed,automatic]
regolith-wm-rofication-ilia/unknown,now 4.11.6-1regolith-noble arm64 [installed,automatic]
regolith-wm-swap-focus/unknown,now 4.11.6-1regolith-noble arm64 [installed,automatic]
regolith-wm-workspace-config/unknown,now 4.11.6-1regolith-noble arm64 [installed,automatic]
xdg-desktop-portal-regolith-x11-config/unknown,now 0.3.5-1regolith-noble arm64 [installed,automatic]
xdg-desktop-portal-regolith/unknown,now 0.3.5-1regolith-noble arm64 [installed,automatic]
xrescat/unknown,now 1.2.1-2-1regolith-noble arm64 [installed,automatic]
